integration/containerd/cri/integration-tests.sh

```diff
@@ -116,6 +116,7 @@ ci_cleanup() {
 
 create_containerd_config() {
 	local runtime="$1"
+	local kata_annotations="$2"
```

Comment:
A mere suggestion: make "$2" optional and defaulting to 0 so that you don't need to change the calls to `create_containerd_config`.

Documenting the parameters would be nice too :)

Reply:
Updated.